modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -0,0 +1,61 @@
+[[salt.rate.limiting]]
+= Rate Limiting
+
+Salt is able to run commands in parallel on a large number of minions.
+This can potentially create large amounts of load on your infrastructure.
+You can use these rate-limiting parameters to control the load in your environment.
+
+These parameters are all configured in the [filename]``/etc/rhn/rhn.conf`` configuration file.
+
+[NOTE]
+====
+Salt commands that are executed from the command line are not subject to these parameters.
+====
+
+
+
+== Batching
+
+There are two parameters that control how actions are sent to clients, one for the batch size, and one for the delay.
+
+When the Salt master sends a batch of actions to the target minions, it will send it to the number of minions determined in the batch size parameter.
+As each minion completes, the action begins on a new minion after the specified delay period.
```

Comment:
```suggestion
As each minion completes, the command begins on a new minion after the specified delay period.
```

Reply:
Not sure if it worth to mention it, but I found this a little bit tricky.
"As each minion completes, the command begins on a new minion after the specified delay period" is true but does not reflect how it behaves completely. This to me gives me a feeling of "the commands on the minions will be executed sequentially as each minion completes".

What actually happens is that, as the minions complete, a next batch is gonna be scheduled to execute the command in new minions. The size of that new batch will be equal to the number of minions that completed the command at the moment the "delay" time expired. Or, better said, the empty places that are left at that moment. (Sorry Lana, I'm not good with rewording as you can see haha)

Reply:
Heh, it's fine. I've had another go :)